src/components/CountryList.vue

```diff
@@ -0,0 +1,16 @@
+<script lang="ts" setup>
+import CountryBoard from "@/components/CountryBoard.vue";
+</script>
+
+<template>
+  <div class="v-countries-list">
+    <CountryBoard />
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.v-countries-list {
+  height: calc(100% - 8.2rem);
```

Comment:
czemu przeliczasz height w taki sposób?

Reply:
wątpię, aby to działalo, chyba ze gdzieś w body albo w html masz ustawione height na jakąś konkretną wartość - inaczej height ustawione jest na auto, a ta wartość znaczy "tyle ile jest contentu" w dużym uproszczeniu. Ogólnie nie widzę sensu w ustawianiu height na pałe, lepiej już manipulować min-height i max-height jeśli masz takie potrzeby w designie